### PR TITLE
Newbranch24

### PR DESCRIPTION
**Pull request by vvikramb**
Added offset values to triggers.

### CODE REVIEW

File: juniper_official/Interfaces/check-lacp-state-netconf.rule [hunk 1]
```diff
@@ -66,7 +66,7 @@
             trigger lacp-interface-distribution-status {
                 synopsis "lacp interface distribution status kpi";
                 description "sets health based on lacp interface distribution status";
-                frequency 200s;
+                frequency 1.11offset;
```
Comment (gvarao) [inline juniper_official/Interfaces/check-lacp-state-netconf.rule:69]: Make offset to 1.1 instead of 1.11

**Review by gvarao — CHANGES_REQUESTED** [review]

File: juniper_official/Interfaces/check-lacp-state-netconf.rule (hunk 1 above)
Comment (vvikramb) [reply juniper_official/Interfaces/check-lacp-state-netconf.rule]: Updated to 1.1 offset in the file.

**Comment by vvikramb** [comment]
jenkins please try a build

**Comment by vvikramb** [comment]
jenkins please retry a build

**Comment by gvarao** [comment]
jenkins please retry a build

**Comment by vvikramb** [comment]
jenkins please retry a build

**Review by gvarao — APPROVED** [review]

**Pull request merged**